<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>1312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Image Series</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="nameGroupBox">
       <property name="title">
        <string>Name</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="nameLineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="imageSeriesGroupBox">
       <property name="title">
        <string>Image Series</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QToolButton" name="imagesToolButton">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click to select image files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>   </string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QTableWidget" name="imagesTableWidget">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>File</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Start</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Stop</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Step</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QTableWidget" name="indVarTableWidget">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Start</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Stop</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Step</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="pathLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="imagesLabel">
          <property name="text">
           <string>Images</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="pathLabel">
          <property name="text">
           <string>Path</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QToolButton" name="pathToolButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click to select path&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>   </string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="indVarLabel">
          <property name="text">
           <string>Independent Variable</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="darkImagFileGroupBox">
       <property name="title">
        <string>Dark Image File</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="darkImageFileLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="darkImageToolButton">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Click to select (or create) dark image file&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>   </string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="orientationGroupBox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This should be moved into the load image series dialog&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="title">
        <string>Orientation</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QCheckBox" name="hFlipCheckBox">
          <property name="text">
           <string>Flip Horizontal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="vFlipCheckBox">
          <property name="text">
           <string>Flip Vertical</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="rotateCheckBox">
            <property name="text">
             <string>Rotate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="cwRotRadioButton">
            <property name="text">
             <string>CW</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="ccwRotRadioButton">
            <property name="text">
             <string>CCW</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionGetImageFiles">
   <property name="text">
    <string>Select image files</string>
   </property>
  </action>
  <action name="actionGetPath">
   <property name="text">
    <string>Select path</string>
   </property>
  </action>
  <action name="actionGetDarkFile">
   <property name="text">
    <string>Select dark file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotateCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>cwRotRadioButton</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>1257</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>1257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rotateCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>ccwRotRadioButton</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>1257</y>
    </hint>
    <hint type="destinationlabel">
     <x>914</x>
     <y>1257</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
